en/docs/security-announcements/security-advisories/2020/WSO2-2019-0665.md

```diff
@@ -24,8 +24,7 @@ cvss: "8.7 (CVSS:3.0/AV:N/AC:L/PR:H/UI:N/S:C/C:H/I:N/A:H)"
 
 
 ### OVERVIEW
-It has identified that Management Console is vulnarable to a XXE attack when updating a EventPublisher.
-
+It has been identified that Management Console is vulnerable to an XXE attack when updating an EventPublisher.
```

Comment:
`An XXE` or `A XXE`?

Reply:
We can use `an` here - https://english.stackexchange.com/questions/8181/does-one-use-a-or-an-before-the-word-x-ray

Reply:
Noted. Thanks for the clarification.